spec/models/spree/gateway_tagging_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+# An inheritance bug made these specs fail.
+# See config/initializers/spree.rb
+shared_examples "taggable" do |parameter|
+  it "uses the given parameter" do
+    expect(subject.tag_list).to eq []
+  end
+end
+
+module Spree
+  describe PaymentMethod do
+    it_behaves_like "taggable"
```

Comment:
why not moving this to the payment_method_spec.rb? That is where I'd go check that if I had to.

Reply:
This spec is actually not about the PaymentMethod but about its descendants. Since I'm testing several classes for the same thing I do it in one place instead of scattering these tests in the spec files of every descendant. We are actually testing Ruby's inheritance here which doesn't belong to one specific class.

Reply:
I added more comments to explain this so that the next developer doesn't have the same question. Thank you for prompting. :+1: :-)